.gitignore

```diff
@@ -3,3 +3,8 @@ build
 .vscode
 .idea
 *.iml
+
+bot
```

Comment:
I would suggest to remove evrything (new) except coverage, since those thing are hard to accidently create here without clear intentions
Not sure what "bot" line is
Like, if u'r building into root directory and not into build directory (default in build script), then sure, something will appear in root dir. But its very intentional thing, and we have build directory just for this
Or yaml and db files - if its result of running bin/debug, than again, that could be placed into build directory
Or we can create debug directory for debug stuff and exclude debug directory as whole and just some specifically named db/config files.

Reply:
**`bot`**: If we build the app with the instructions in the README we end up with `bot` binary in the root directory. Yes, we can say that README should be updated (and it should) but I don't see any harm to still have this in the `.gitignore` file. 

**`camino-messenger-bot.yaml`**: This is the config file mentioned in the README. So, if you run the bot as it is, we end up this created in the root directory. It's easy enough to push this accidentally and it contains private keys. IMO, we should also add the new example config files (without the .example extension).

**`mautrix.db`**: Again, same as above. Created if you follow the README.

Reply:
I'd suggest to update readme. It doesn't make sense to tell people in readme to build bin with go command, while we have our own build script that does that and maybe some additional actions (and it most likely will, like, passing git commit to compiler for version printout)

Reply:
I've updated the README.me file with instructions to use the build.sh to build the bot.